config/locales/forms/2023/sales/household_situation.en.yml

```diff
@@ -0,0 +1,50 @@
+en:
+  forms:
+    2023:
+      sales:
+        household_situation:
+          prevten:
+            page_header: ""
+            check_answer_label: "Buyer 1’s previous tenure"
+            hint_text: ""
+            question_text: "What was buyer 1’s previous tenure?"
+
+          last_accommodation:
+            page_header: ""
+            ppcodenk:
+              check_answer_label: "Buyer 1’s last settled accommodation"
+              hint_text: "This is also known as the household’s 'last settled home'"
+              question_text: "Do you know the postcode of buyer 1’s last settled accommodation?"
+            ppostcode_full:
+              check_answer_label: "Postcode of buyer 1’s last settled accommodation"
+              hint_text: ""
+              question_text: "Postcode"
+
+          last_accommodation_la:
+            page_header: ""
+            previous_la_known:
+              check_answer_label: "Local authority of buyer 1’s last settled accommodation"
+              hint_text: "This is also known as the household’s 'last settled home'"
+              question_text: "Do you know the local authority of buyer 1’s last settled accommodation?"
+            prevloc:
+              check_answer_label: "Local authority of buyer 1’s last settled accommodation"
+              hint_text: ""
+              question_text: "Select a local authority"
```

Comment:
When there are 2 questions on a page, I've just been defining them the same as 1 question per page, to get the default copy_key behaviour. Is there pros/cons to doing it one way over the other?

Reply:
Assuming that we don't change what's on the page / the order of things on the page later, I guess it doesn't make much difference.

I think I find this slightly more logical, in that it wouldn't be possible to set page_header from the second question so it either has a useless setting or looks weird without explanation by missing that setting. (It fits with how type / othtype ended up in the setup section, although that one has the additional complication of there being three pages/question sets)

Reply:
Okay, I think it makes sense. I'll adjust my PR to be consistent with this. Thank you!